docs/spec/v1beta2/providers.md

```diff
@@ -124,6 +124,7 @@ The supported alerting providers are:
 | [Slack](#slack)                                         | `slack`          |
 | [Telegram](#telegram)                                   | `telegram`       |
 | [WebEx](#webex)                                         | `webex`          |
+| [PagerDuty](#pagerduty)                                 | `pagerduty`      |
```

Comment:
Please move PagerDuty right before Prometheus. Also in the section below with examples.